qmd/sets.qmd

```diff
@@ -22,119 +22,351 @@ source("R/qmd-utils.R")
   for (set_ID in unique_set_IDs) {
-    set_ID_info <- parts_table[parts_table[[parts_sheet_column_names$part_ID_column_name]] == set_ID, ]
+    set_ID_info <-
+      parts_table[parts_table[[parts_sheet_column_names$part_ID_column_name]] == set_ID,]
     # Handle no rows found for set_ID in parts
-    if(nrow(set_ID_info)<1){
+    if (nrow(set_ID_info) < 1) {
       warning(glue::glue('Set ID:{set_ID} has no data in parts sheet and is skipped'))
       next()
     }
-    set_ID_part_label <- set_ID_info[[parts_sheet_column_names$part_label_column_name]]
+    set_ID_part_label <-
+      set_ID_info[[parts_sheet_column_names$part_label_column_name]]
     # Create set_ID subsection
-    display_content <- verify_and_append_content(
-      display_content,
+    set_display <- verify_and_append_content(
+      set_display,
       set_ID_part_label,
       glue::glue(
         'Set ID: {set_ID} has no valid {parts_sheet_column_names$part_label_column_name}'
       ),
-      glue::glue('
-      
-                 ### {set_ID_part_label} {{ #{set_ID} }}
-                 
-                 '
-      ),
-      glue::glue('
-      
-                 ### {set_ID}
-                 
-                 '
-      ))
+      glue::glue('\n\n
+      ### {set_ID_part_label} {{ #{set_ID} }}'),
+      glue::glue('\n\n
+                 ### {set_ID}')
+    )
+    # Declare additional setID display
+    set_ID_part_description <- set_ID_info[[parts_sheet_column_names$part_description_column_name]]
+    set_ID_part_instruction <- set_ID_info[[parts_sheet_column_names$part_instruction_column_name]]
+    set_ID_part_status <- set_ID_info[[parts_sheet_column_names$part_status_column_name]]
+    set_ID_part_first_released <- set_ID_info[[parts_sheet_column_names$part_first_release_column_name]]
+    set_ID_part_last_updated <- set_ID_info[[parts_sheet_column_names$part_last_updated_column_name]]
+    
+    # Verify display elements
+    verify_input(set_ID_part_description,
+                 glue::glue('Part description is missing for {set_ID}'))
+    verify_input(set_ID_part_instruction,
+                 glue::glue('Part instruction is missing for {set_ID}'))
+    verify_input(set_ID_part_status,
+                 glue::glue('Part status is missing for {set_ID}'))
+    verify_input(set_ID_part_first_released,
+                 glue::glue('Part first released is missing for {set_ID}'))
+    verify_input(set_ID_part_last_updated,
+                 glue::glue('Part last updates is missing for {set_ID}'))
+    
+    set_display <- glue::glue(
+      '{set_display}
+<a href=\"/parts.html#{set_ID}\">({set_ID})</a>{set_ID_part_description} \\
+{if(set_ID_part_instruction %!=na% constants$dictionary_missing_value_replacement) set_ID_part_instruction else ""} </br>
+Status: {set_ID_part_status}. First Released: {set_ID_part_first_released}. \\
+Last updated: {set_ID_part_last_updated}.'
+    )
+    
     # Create list to hold a values for html list
     list_values <- list()
```

Comment:
Should be named more specific, right now I don't know what list values its supposed to hold. For example, `set_cat_list_values`.